### PR TITLE
Standard nginx

### PR DESCRIPTION
**Pull request by melindaloubser1**
Use the default nginx image for the nginx deployment and adding config files & entrypoint script as volume mounts
Addresses https://github.com/RasaHQ/rasa-x/issues/3428

### CODE REVIEW

**Comment by melindaloubser1** [comment]
@tczekajlo since https://github.com/RasaHQ/rasa-x/issues/3428#issuecomment-805889815
the custom entrypoint is not necessary.

File: charts/rasa-x/Chart.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 
 version: "1.8.1"
 
-appVersion: "0.35.0"
+appVersion: "0.37.0"
```
Comment (melindaloubser1) [inline charts/rasa-x/Chart.yaml:6]:
```suggestion
appVersion: "0.38.0"
```

File: charts/rasa-x/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ apiVersion: v2
 
 version: "1.8.1"
```
Comment (melindaloubser1) [inline charts/rasa-x/Chart.yaml:4]:
```suggestion
version: "1.8.2"
```

**Comment by melindaloubser1** [comment]
@tczekajlo What else needs to happen to this to make it mergeable?

File: charts/rasa-x/templates/nginx-deployment.yaml [hunk 3]
```diff
@@ -34,15 +34,17 @@ spec:
...
-        {{- if .Values.nginx.command }}
         command:
+        {{- if .Values.nginx.command }}
           {{- toYaml .Values.nginx.command | nindent 10 }}
         {{- end }}
-        {{- if .Values.nginx.args }}
         args:
+        {{- if .Values.nginx.args }}
           {{- toYaml .Values.nginx.args | nindent 10 }}
+        # {{- else }}
```
Comment (tczekajlo) [inline charts/rasa-x/templates/nginx-deployment.yaml:46]: Can we remove it?

**Review by tczekajlo — APPROVED** [review]
@melindaloubser1 You have to bump up the chart version and then I think it should be ready

File: charts/rasa-x/values.yaml [hunk 4]
```diff
@@ -472,9 +472,11 @@ nginx:
   # override the default arguments to run in the container
   args: []
   # name of the nginx image to use
-  name: "rasa/nginx"
+  name: "nginx"
   # tag refers to the nginx image tag (uses `appVersion` by default)
-  tag: ""
+  tag: "1.19"
+  # custom config map containing  nginx.conf, rasax.nginx.template
```
Comment (tczekajlo) [inline charts/rasa-x/values.yaml:478]:
```suggestion
  # custom config map containing nginx.conf, rasax.nginx.template
```

File: charts/rasa-x/nginx-config-files/nginx.conf [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+    access_log /dev/stdout;
+
+    client_body_temp_path  "/etc/nginx/client_body" 1 2;
+    proxy_temp_path        "/etc/nginx/proxy" 1 2;
+    fastcgi_temp_path      "/etc/nginx/fastcgi" 1 2;
+    scgi_temp_path         "/etc/nginx/scgi" 1 2;
+    uwsgi_temp_path        "/etc/nginx/uwsgi" 1 2;
+
+    sendfile        on;
+    #tcp_nopush     on;
```
Comment (tczekajlo) [inline charts/rasa-x/nginx-config-files/nginx.conf:28]:
```suggestion
```

**Comment by melindaloubser1** [comment]
@tczekajlo I'm going to add back the commented-out ssl conf like for the docker-compose install, just so they are consistent, before merging.

**Pull request merged**